packages/vats/src/mintHolder.js

```diff
@@ -17,19 +17,25 @@ import { makeDurableIssuerKit } from '@agoric/ertp';
  * @param {unknown} _privateArgs
```

Comment:
not your change but might be worth tightening this up
```suggestion
 * @param {undefined} _privateArgs
```